doc/classes/KinematicBody3D.xml

```diff
@@ -132,6 +135,8 @@
 			</argument>
 			<argument index="6" name="infinite_inertia" type="bool" default="true">
 			</argument>
+			<argument index="7" name="delta" type="float" default="0.0">
+			</argument>
 			<description>
 				Moves the body while keeping it attached to slopes. Similar to [method move_and_slide].
 				As long as the [code]snap[/code] vector is in contact with the ground, the body will remain attached to the surface. This means you must disable snap in order to jump, for example. You can do this by setting [code]snap[/code] to [code](0, 0, 0)[/code] or by using [method move_and_slide] instead.
```

Comment:
Documentation for the `delta` parameter should be added here as well.

Reply:
@Calinou I'm not sure I understand. The last line of the description does include details about the `delta` parameter.

Reply:
@Calinou I purposely didnt add it here because it says "similar to method `move_and_slide`" and doesn't detail any of the other parameters that `move_and_slide` has. It just explains the addition of the snap bit.